### PR TITLE
Adding Files

### PR DESCRIPTION
**Pull request by zaidansari42**
I have added my files and updated my ReadMe

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dice part 2</title>
```
Comment (murtazshaikh) [inline index.html]: set proper title eg: 3D Box Hover Effect

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  background: #b3886e;
+}
+
+.container:hover .text {
+  transform: rotateX(-360deg);
+}
+
+.container:hover .text:last-child {
+  transform: rotateX(810deg);
+}
```
Comment (murtazshaikh) [inline styles.css:92]: add responsiveness

**Review by murtazshaikh — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (zaidansari42) [reply index.html]: Okay

**Review by syedabdullahali — APPROVED** [review]

**Review by ManSOorcode — APPROVED** [review]

**Review by syedabdullahali — APPROVED** [review]

**Review by anask601 — COMMENTED** [review]
merge this into main and your branch name should be meaningful

File: styles.css (hunk 2 above)
Comment (zaidansari42) [reply styles.css:92]: Working on it

**Review by zaidansari42 — COMMENTED** [review]
Made All the changes as per the suggestions and review. Project Ready to be Deployed

**Comment by zaidansari42** [comment]
> merge this into main and your branch name should be meaningful

Okay will keep in mind this in future

**Pull request merged**